tools/azure-pipelines/build-python-wheels.yml

```diff
@@ -24,7 +24,6 @@ jobs:
       # Compile
       - script: |
           ${{parameters.environment}} ./tools/ci/compile.sh
-          ./tools/azure-pipelines/create_build_artifact.sh
```

Comment:
If python no longer relies on this then a number of exclusions in `create_build_artifact.sh` could probably be removed. Maybe even all of them.

Reply:
`build-python-wheels.yml` never relied on this script. I don't fully understand this comment.
The excludes in `create_build_artifact.sh` are for reducing the file size because those files are not needed?!

Reply:
(note: the comment was on an intermediate commit, not the final version of the file )

Reply:
The excludes are adding files _back_, increasing the file size.

Reply:
Ah, I see. I will look into this :)

Reply:
Initial tests suggest that we can really remove all exclusions :)